core/base/document.php

```diff
@@ -1369,6 +1370,101 @@ public function __construct( array $data = [] ) {
 		parent::__construct( $data );
 	}
 
+	/**
+	 * Get document settings usage.
+	 *
+	 * @return array
+	 */
+	public function get_usage() {
```

Comment:
It's not a part of 3.6.0. Please ensure only 3.6.0 is merged in the PR

Reply:
Handled.